docs/docs/reference/signing.md

```diff
@@ -15,27 +15,37 @@ Some of these signing methods are inherited from the amazing Sigstore community
-| Ephemeral ([EJBCA](https://github.com/Keyfactor/ejbca-ce)) | Connect your EJBCA instance to your Chainloop deployment using [these settings](https://github.com/chainloop-dev/chainloop/blob/main/deployment/chainloop/templates/controlplane/ejbca_ca.secret.yaml). Omit `--key` when pushing the attestation. | Use your verification material with `--cert` and `--cert-chain` options in the `wf run describe` command. <br/>See [bundles](#bundles)                      |
-| [SignServer](https://www.signserver.org/)                  | You can sign with your instance of SignServer with `--key signserver://host/worker`                                                                                                                                                                | Both signing certificate and chain must be provided out of band. Use `--cert signingcert.pem --cert-chain root.pem` to verify. <br/>See [bundles](#bundles) | 
-
-#### Bundles 
-When signing with a verification method that supports it (like ephemeral certificates), you can download the verification material, to be used later during the verification process.
+### Signing with a local key
+These methods require setting up a key and/or KMS authentication in the local environment (laptop, CI system ...).
+
+| Method            | Signing (`chainloop att push`)                                                                         | Verifying (`chainloop wf run describe --verify true`)                      |
+|-------------------|--------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------| 
+| Cosign key        | `--key cosign.key`                                                                                     | `--key cosign.pub`                                                         |
+| KMS               | `--key awskms://<KeyID>`                                                                               | `--key awskms://<KeyID>`                                                   | 
+| PKCS#11           | `--key pkcs11://<KeyId>`                                                                               | `--key pkcs11://<KeyId>`                                                   |
+| Kubernetes secret | `--key k8s://<namespace>/<secretName>` (where `cosign.key` and `cosign.password` secrets are expected) | `--key k8s://<namespace>/<secretName>` (where `cosign.pub` is expected)    |
+| Gitlab secret     | `--key gitlab://<project>` (it will look for `COSIGN_PRIVATE_KEY`, `COSIGN_PASSWORD` variables)        | `--key gitlab://<project>` (it will look for `COSIGN_PUBLIC_KEY` variable) |
+
+### Keyless signing
+These methods don't require any special setup in the client. You must make sure you get the CA certificate chain out-of-band, as it will be required for the verification command.
```

Comment:
> You must make sure you get the CA certificate chain out-of-band, as it will be required for the verification command.

I am not sure I understand what this means. It might be easier to explain that this requires configuration in controlplane during deployment, and point to the Hel chart section maybe?

Reply:
It's related to the verification in the CLI. You always need the root certificate (no matter the CA you are using), which must be provided by your organization.